connectors/src/api/webhooks/webhookSlack.ts

```diff
@@ -33,7 +34,11 @@ type SlackWebhookResBody =
   | ConnectorsAPIErrorResponse;
 
 const _webhookSlackAPIHandler = async (
-  req: Request<null, SlackWebhookResBody, SlackWebhookReqBody>,
+  req: Request<
```

Comment:
This API handler was not returning the proper API error objects because the type of the Request object previously returned was not compatible with the type needed by `apiError()`. This change fixes it. I will update all the other returns of this handler in a different PR.